partiql-value/src/lib.rs

```diff
@@ -1674,6 +1698,66 @@ mod tests {
             Value::from(false),
             NullableEq::eq(&Value::from(true), &Value::from(false))
         );
+        // Container examples from spec section 7.1.1 https://partiql.org/assets/PartiQL-Specification.pdf#subsubsection.7.1.1
```

Comment:
Tests from L1701-1760 and L1855-1915 are taken from [spec section 7.1.1](https://partiql.org/assets/PartiQL-Specification.pdf#subsubsection.7.1.1) and aren't affected by this PR's changes.